src/components/OverlayManager/Login/RegisterForm.tsx

```diff
@@ -51,7 +52,14 @@ const RegisterForm: React.FC<{ hide: () => void }> = ({ hide }) => {
             onSubmit={(event, { email, password }) => {
               event.preventDefault();
               const redirectUrl = `${location.origin}${paths.accountConfirm}`;
```

Comment:
Just a nitpick, but it should be `url-join` used here rather than simple concatenation